bin/add_ancillary.py

```diff
@@ -0,0 +1,457 @@
+"""
+Copyright (C) 2012 by Forschungszentrum Juelich GmbH
```

Comment:
Please update!

Reply:
To what though? I am an absolute idiot when it comes to copyrights.
Is this still a copyright by FZJ? Is it a copyright by MSS? Is it a copyright by you and me?

Reply:
Adopt year and authors. Maybe Reimar has additional comments. See his license comment. I'd handle this exactly as we did with MSS.